### DISCUSSION TITLE
This version of the Android SDK (10.0) is not supported by Uno Platform

### DISCUSSION THREAD

**Discussion by stefanolson** [unoplatform/uno#9184] [Q&A] [answered]
### Current behavior

My class library project, that was previously running Uno 3.4, now won't build with an error when using Uno 4.4:
Error: This version of the Android SDK (10.0) is not supported by Uno Platform. You must change the "Compile using Android version:" field in the android project property with at least version 11.0

In my android project, if open the project on the first page it says android 11 under "compile using android version (target framework)" and on the manifest page also says that for target android version:
![image](https://user-images.githubusercontent.com/1751192/176679750-666062de-7483-4547-9449-531a6614b2c4.png)

In my project file for the class library (which is what is failing to build) and has the following things listed:
`<TargetFrameworks>uap10.0.18362;netstandard2.0;xamarinios10;xamarinmac20;monoandroid10.0</TargetFrameworks>`

Not sure if that needs to be changed as well? monoandroid11.0 doesn't make any difference. The documentation here: https://platform.uno/docs/articles/migrating-from-previous-releases.html doesn't say anything about it.

### Expected behavior

The class library project should build, as the target framework for android appears to be correct

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the report. [Upgrading to Uno 4.1](https://platform.uno/docs/articles/migrating-from-previous-releases.html#android-12-support) and later requires to upgrade apps and libraries Android 11 and later.

In your case, you'll need to upgrade your library's `<TargetFrameworks>` from `monoandroid10.0` to `monoandroid11.0`, or `monoandroid12.0` if you want to gain 1.5 more years of support (Android 12 SDK support will likely be removed in October 2023).

**Comment by stefanolson** [comment]
Thanks. I've tried that again (I tried twice before, but this time I did a rebuild all), and it now works. So onto finding the next issue. Thanks!
